### PR TITLE
Plugin Api

### PR DESCRIPTION
**Pull request by castelbuilder123**
Ps: you should really add me on skype =]

### CODE REVIEW

File: CoCSharp.Client/RemoteServer.cs [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Net;
+using System.Net.Sockets;
+
+namespace CoCSharp.Client
+{
+    class RemoteServer
```
Comment (FICTURE7) [inline CoCSharp.Client/RemoteServer.cs]: Whats up with this crap?

Comment (castelbuilder123) [inline CoCSharp.Client/RemoteServer.cs]: The Network library server

Comment (FICTURE7) [inline CoCSharp.Client/RemoteServer.cs]: It shouldn't be there.

Comment (castelbuilder123) [inline CoCSharp.Client/RemoteServer.cs]: yeah look at the commit title which added it: wip stuff i needed to add for travis-ci to work

Comment (FICTURE7) [inline CoCSharp.Client/RemoteServer.cs]: Yeah it was in your commit, which was then added to the branch, which meant that we had some random useless class in the source.

**Pull request merged**